src/server.ts

```diff
@@ -305,11 +305,18 @@ wss.on('connection', async (ws: Server.Ws) => {
       }
 
       case MESSAGES.SUBSCRIBE_ADDRESS: {
-        if (data.params.addresses && data.params.addresses.length > 0) {
-          for (const addressInput of data.params.addresses) {
-            if (!addressesSubscribed[clientId].includes(addressInput)) {
-              addressesSubscribed[clientId].push(addressInput);
-            }
+        const { addresses, options } = { options: {}, ...data.params };
```

Comment:
small nitpick, can we please add `cbor` param straight to method's `params` instead of encapsulating it in `options` object?

i.e. the message would look like this:
```
{
    "id": 0,
    "command": "SUBSCRIBE_ADDRESS",
    "params": {
        "cbor": true,
        "addresses": [
            "..."
        ]
    }
}

```

Reply:
3d86ac8 should solve this